### PR TITLE
Fix PEP-8 violations reported by flake8 on asv folder

### PR DESCRIPTION
**Pull request by LucyJimenez**
Check flake8 and fix issues pep8 related on asv/asv folder for:

1. statistics.py
2. runner.py
3. results.py
4. repo.py
5. main.py

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ jobs:
...
         run: pip install flake8
+      
+      - name: flake8 all repo only show erros
+        run: flake8 . || true # preventing CI failing while flake8 issues exist
 
-      - name: flake8
-        run: flake8 . || true # preventing CI failing while flake8 issues exist
\ No newline at end of file
+      - name: flake8 validating correct file(s)
+        run: flake8 asv/step_detect.py asv/runner.py asv/results.py asv/repo.py asv/main.py
```
Comment (datapythonista) [inline .github/workflows/ci.yml:19]: Are all the files in this PR added here? Seems like some are missing, is there a reason for it?

File: asv/util.py [hunk 2]
```diff
@@ -531,7 +543,7 @@ def get_content(header=None):
     if env and WIN and sys.version_info < (3,):
         # Environment keys and values cannot be unicode
         def _fix_env(s):
-            return s.encode('mbcs') if isinstance(s, unicode) else s
+            return s.encode('mbcs') if isinstance(s, unicode) else s # noqa
```
Comment (datapythonista) [inline asv/util.py:546]: Another space is missing before the hash. Also, we need the error we are ignoring after noqa.

File: asv/util.py [hunk 3]
```diff
@@ -1127,9 +1147,9 @@ def format_text_table(rows, num_headers=0,
 
 def _datetime_to_timestamp(dt, divisor):
     delta = dt - datetime.datetime(1970, 1, 1)
-    microseconds = (delta.days * 86400 + delta.seconds) * 10**6 + delta.microseconds
+    microseconds = ((delta.days * 86400 + delta.seconds) * 10**6 + delta.microseconds)
```
Comment (datapythonista) [inline asv/util.py:1150]: Spaces around the ** operator

**Review by datapythonista — COMMENTED** [review]
Looks good, added few comments. For next time, try to open smaller PRs.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (LucyJimenez) [reply .github/workflows/ci.yml:19]: Thank you, Marc. You are right! I'm going to include all the respective files.

File: asv/statistics.py [hunk 4]
```diff
@@ -387,7 +388,7 @@ def binom_pmf(n, k, p):
     logp = math.log(p)
     log1mp = math.log(1 - p)
     return math.exp(lgamma(1 + n) - lgamma(1 + n - k) - lgamma(1 + k)
-                        + k*logp + (n - k)*log1mp)
+                    + k*logp + (n - k)*log1mp)
```
[the remark this answers is not in the document]
Comment (LucyJimenez) [reply asv/statistics.py:391]: It was my mistake. I'm working on fixing it!

File: asv/util.py (hunk 2 above)
Comment (LucyJimenez) [reply asv/util.py:546]: Done!

File: asv/util.py (hunk 3 above)
Comment (LucyJimenez) [reply asv/util.py:1150]: I don't think spaces are foolish, and flake8 doesn't point to it as a mistake.

**Comment by datapythonista** [comment]
CI is red, looks like you're breaking the tests. This PR is too big, maybe better to start by splitting it like in 3, one for util.py, one for step_detect.py which are quite large files, and one for the rest. This will help find the problem, as well as making it easier to review, even if it takes few iterations.

**Comment by LucyJimenez** [comment]
@datapythonista Thank you for your advice. Now, this PR is shorter than before, and de CI is working!

**Review by datapythonista — APPROVED** [review]
Thanks @LucyJimenez, really nice.

We'll merge it after the release, but looks perfect.

**Comment by datapythonista** [comment]
Thanks @LucyJimenez

**Pull request merged**